project-virus/scenes/levels/play_button.gd

```diff
@@ -0,0 +1,15 @@
+extends Button
+
+@export var transition_color: ColorRect # Adjust the path if necessary
+var fade_duration = 1.0 # Duration for fade in/out
+
+func _on_pressed():
```

Comment:
We have a SceneTransitioned component now so we can look to replace this. But works for now